content/_data/changelogs/weekly.yml

```
@@ -5223,6 +5223,74 @@
   # pull: 4303 too minor (PR title: Use actual logger to record Jenkins home and where it was found)
   # pull: 4313 (PR title: [JENKINS-58625] Cancel older builds of a given branch)
 
+- version: '2.203'
+  date: 2019-11-05
+  changes:
+
+# Major RFE
+  - type: major rfe
+    issue: 19887
+    message: |-
+      Allow time zone to be set on a per user basis
+    pull: 4113
+    authors:
+      - silent-snowman
+
+  # RFEs
+
+  - type: rfe
+    message: |-
+      Minor improvements to the logging UI
+    pull: 4305
+    authors:
+      - daniel-beck
+  - type: rfe
+    message: |-
+      Update the Plugin Manager Updates View with more Info about incompatible dependencies and change CSS classes
+    pull: 4299
+    authors:
+      - andipabst
+  - type: rfe
+    issue: 17459
+    message: |-
+      Clicking red/yellow/green icons link to the build Console
```

"But this already existed?"

Also, they're not green.

```suggestion
      Build status balls on the build trend page now link to the respective build's console output.
```